src/components/markdown-renderer/replace-components/abc/abc-frame.tsx

```diff
@@ -24,7 +27,7 @@ export const AbcFrame: React.FC<AbcFrameProps> = ({ code }) => {
         imp.renderAbc(actualContainer, code, {})
       })
       .catch(() => {
-        console.error('error while loading abcjs')
+        log.error('error while loading abcjs')
```

Comment:
```suggestion
        log.error('Error while loading abcjs', error)
```

(add the error to the catch too)